lib/ads/interstitial_ad_manager.js

```diff
@@ -598,6 +609,27 @@ shaka.ads.InterstitialAdManager = class {
   }
 
 
+  /**
+   * @private
+   */
+  cuepointsChanged_() {
+    /** @type {!Set.<!shaka.extern.AdCuePoint>} */
+    const cuePoints = new Set();
+    for (const interstitial of Array.from(this.interstitials_)) {
```

Comment:
Is there any reason to use `Array.from()` here? You can do a for-of loop on a set, I thought.

Reply:
Done!